nitro-protocol/contracts/LedgerFinancingApp.sol

```diff
@@ -16,12 +16,12 @@ contract LedgerFinancingApp is IForceMoveApp {
     }
 
     struct InterestAppData {
-        // a per-day simple interest rate (daily percentage yield), as a fraction.
-        // ie, 1% per day is represented with dpyNum = 1 and dpyDen = 100.
-        //
-        // Lower numbers (eg, fraction in simplest terms) produce least risk of overflow.
-        uint128 dpyNum;
-        uint128 dpyDen;
+        // the per-block interest rate, expressed as the denominator of a fraction
```

Comment:
Nit: Is there potential for confusion here? `bpy` is really an inverse interest rate, so could be named as such (`interestPerBlockDivisor`, say)

Reply:
57bebfdb